config.yaml.dist

```diff
@@ -31,6 +31,16 @@ clusterNetwork:
   serviceDomainName: ""
   # a nodePort range to reserve for services (default: 30000-32767)
   nodePortRange: ""
+  # CNI plugin choise
+  cni:
```

Comment:
Mention that CNI choice is not respected at the upgrade time and that CNI plugin is not updated on upgrade.